src/app/create-profile/page.tsx

```diff
@@ -0,0 +1,68 @@
+'use client';
+
+import { UUID } from 'crypto';
+import { useState } from 'react';
+import { insertProfile } from '../../api/supabase/queries/profiles';
+import { Profile } from '../../types/schemaTypes';
+import styles from '../page.module.css';
+
+export default function Profile() {
+  const [firstName, setFirstName] = useState<string>('');
+  const [lastName, setLastName] = useState<string>('');
+
+  const handleInsert = async () => {
+    if (!firstName && !lastName) return;
+    const newProfile: Profile = {
+      // hardcoded values for now
+      user_id: crypto.randomUUID() as UUID,
+      roles: ['blah'],
+      languages: ['blah', 'blah'],
+      accreditations: ['blah'],
+      hours_per_week: 40,
+      immigration_law_experience: '5 years',
```

Comment:
since we're changing the schema, we will need to change the law_experience to an int!

Reply:
immigration_law_experience should be "advanced", "intermediate", or "beginner" -- the "5 years" was a placeholder so i could find it in the supabase table!